_pages/dev/performance/performance-best-practices.md

```diff
@@ -41,6 +41,34 @@ You can improve the performance of your storefront application with the followin
 - Maintain a second level (L2) cache for all of the site assets, and also for the HTTP requests that can be cached, such as OCC calls for anonymous users that do not change very often.
 - Ensure you have an appropriate browser cache policy for all of the assets of the page. Browser caching saves a lot of bandwidth and improves page load time (both initial and subsequent) when it is properly set and tuned.
 
+## Image Format Recommendations
+
+You can improve the performance of your storefront by following these image format recommendations:
+
+- Using WebP or AVIF (AV1 Image File Format) can drastically reduce the size of your images, which can improve the performance of your storefront application.
+  - Check browser support for WebP and AVIF formats before using them.
+  - Provide fallback options for browsers that do not support WebP or AVIF formats.
+- Select the correct compression level for your images to balance image quality and file size.
+  - Use lossy compression for photographs and images with gradients.
+  - Use lossless compression for images with text, sharp edges, or when quality cannot be compromised.
+- Use CDN services that support WebP and AVIF formats to deliver images to users.
+- Use vector graphics (SVG) for simple images and icons, which can be scaled without losing quality and have smaller file sizes compared to raster images.
+- Use CSS techniques to create simple shapes and icons instead of using images. This can reduce the number of image files needed and improve loading times.
+
+## CDN Recommendations
+
+Using a CDN can significantly improve the performance of your storefront by distributing your content across multiple servers located around the world. Here are some best practices for using a CDN:
+
+- Choose a Reliable CDN Provider: Select a CDN provider that offers good performance, reliability, and coverage in the regions where your users are located.
+- Cache Static Assets: Configure your CDN to cache static assets such as images, CSS, JavaScript files, and fonts. This reduces the load on your origin server and speeds up content delivery to users.
```

Comment:
I'd say to maximixe performance, not only static assets should be cached, but also the dynamic server-side-rendered routes (HTML) should be cached.